primer/source/includes/examples-remove.yaml

```diff
@@ -208,7 +207,7 @@ operation:
     language: javascript
     code: |
       var removeRestaurants = function(db, callback) {
-         db.collection('restaurants').remove( {}, function(err, results) {
+         db.collection('restaurants').removeMany( {}, function(err, results) {
```

Comment:
Hey Christian -- 
this example, it's not deleteMany?  So, in node, we have deleteOne, deleteMany, and removeMany?